driver-core/src/test/java/com/datastax/driver/core/exceptions/FunctionExecutionExceptionTest.java

```diff
@@ -23,6 +24,7 @@
 import java.util.Collection;
 
 @CassandraVersion(major = 2.2)
+@CCMConfig(config = "enable_user_defined_functions:true")
```

Comment:
Isn't `@CCMConfig` enough to make the test pass under all versions of C* ? I believe `@CassandraVersion` is already filtering out this test for C* < 2.2, although I don't see any harm in adding an extra check for invalid configurations in CCMTestsSupport.